models/prices/prices_usd_latest_old.sql

```diff
@@ -0,0 +1,22 @@
+{{ config(
+        schema='prices',
+        alias = alias('usd_latest_old'),
+        tags= ['static', 'dunesql'],
+        materialized = 'incremental',
+        file_format = 'delta',
+        incremental_strategy = 'merge',
+        )
+}}
+
+SELECT
+  pu.blockchain
+, pu.contract_address
+, pu.decimals
+, pu.symbol
+, max(pu.minute) as minute
+, max_by(pu.price, pu.minute) as price
+FROM {{ source('prices', 'usd') }} pu
+{% if is_incremental() %}
+    WHERE minute >= date_trunc('day', now() - interval '2' day)
```

Comment:
How does this work? If it's an incremental run, then only compute this subset, and try to merge it into the table? How does it know to use blockchain, contract_address, decimals and symbol as a key, and minute, price as a value?

Reply:
was a bit too quick here, I forgot to add `unique_key = [...]` in the config

Reply:
yep, this is right